src/tss_callbacks.c

```diff
@@ -0,0 +1,118 @@
+/*
+ * This file and its contents are licensed under the Apache License 2.0.
+ * Please see the included NOTICE for copyright information and
+ * LICENSE-APACHE for a copy of the license.
+ */
+
+#include <postgres.h>
+
+#include <fmgr.h>
+#include <executor/instrument.h>
+
+#include "guc.h"
+#include "tss_callbacks.h"
+
+#define TSS_CALLBACKS_VAR_NAME "tss_callbacks"
+#define TSS_CALLBACKS_VERSION 1
+
+typedef void (*tss_store_hook_type)(const char *query, int query_location, int query_len,
+									uint64 query_id, uint64 total_time, uint64 rows,
+									const BufferUsage *bufusage, const WalUsage *walusage);
+typedef bool (*tss_enabled)(int level);
```

Comment:
```suggestion
typedef bool (*tss_enabled_hook_type)(int level);
```

to be consistent with `tss_store_hook_type`